### DISCUSSION TITLE
Random Windows 10 Big Search Box Appeared

### DISCUSSION THREAD

**Discussion by TimMarvelHero** [valinet/ExplorerPatcher#2837] [General]
I randomly just restarted Explorer Patcher, and it showed up the big search box. Wasn't it removed in 23H2 because Microsoft removed the code in it? Or is this a new update, because I updated about an hour ago.
<img width="298" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/152800265/7de535ef-2d17-4319-8c6d-ef53e6f315f6">

**Comment by Amrsatrio** [comment]
Part of the EP update.

**Reply by TimMarvelHero** [reply]
great update!

**Reply by haydengriz** [reply]
Finally!

**Comment by Adorezuki** [comment]
Well it has the bug when you type something in search box it will gone.

**Reply by TimMarvelHero** [reply]
Yeah same thing.
